app/scripts/controllers/preferences.js

```diff
@@ -403,6 +419,16 @@ export default class PreferencesController {
     return Promise.resolve(tokens);
   }
 
+  async updateTokenType(tokenAddress) {
+    const { tokens } = this.store.getState();
+    const previousIndex = tokens.findIndex((token) => {
```

Comment:
Minor, but thoughts on using `tokenIndex` instead of `previousIndex`?  `previous` makes me feel like it's going to or has changed.

Reply:
Also makes sense. I'll make this change.